catlin/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.14 as BUILD
+
+COPY . /build
+WORKDIR /build
+RUN GOOS=linux GARCH=amd64 CGO_ENABLED=0 \
+    go build -mod=vendor -o catlin ./cmd/catlin
+
+FROM scratch
```

Comment:
nit: any reason to use `scratch` ? long-term we may need to want to debug a container running this image, and/or might need to stuff like certificates, …
Any way, no need to change this now

Reply:
> @SM43 any reason to vendor the dependencies ? we do have multiple go modules in the repository, and so far we didn't use `vendor` for them 😅
> If we want to go that way (vendoring), we would need a script to make sure the `vendor` folder is up-to-date. I would rather not vendor anything, at least not while `catlin` is in this repository.

Lets go without vendor then :D ... I am not a big fan of vendor either but the image build is super fast 😝